config/commandline_test.go

```diff
@@ -65,7 +66,7 @@ func TestParseFlags(t *testing.T) {
 
 	assert.Equal(t, "cluster2", upstream2.Label)
 	assert.Equal(t, "localhost:7002", upstream2.UpstreamConfigHost)
-	assert.Equal(t, 10, upstream2.MinPoolSize)
+	assert.Equal(t, 1, upstream2.MinPoolSize)
```

Comment:
minPoolSize should be 1 and maxPoolSize should be 10. Earlier code had a test failure. I have added the second assertion to the test as well.

Reply:
@anujjamwal It should be 10. The change you have on line 39 to defer `assert.NoError(t, os.Unsetenv(minPoolEnvVar))` is actually unsetting the environment variable because the arguments are being evaluated on that line.

Reply:
During parsing, the value is being set with default values at https://github.com/coinbase/redisbetween/blob/e5345604312343cbfba32a99266faeb8edcc5d86/config/config.go#L138

```golang
MaxPoolSize:        getIntParam(params, "maxpoolsize", 10),
MinPoolSize:        getIntParam(params, "minpoolsize", 1),
```

This is code is being used as is.